app/views/historic_appointments/_role_appointment.html.erb

```diff
@@ -1,9 +1,10 @@
-<%= content_tag_for :li, role_appointment, class: 'person person-excerpt' do %>
-  <div class="inner">
-    <div class="image-holder">
-      <%= link_to_if role_appointment.has_historical_account?, role_appointment.person.image, historic_appointment_path(role.historic_param, role_appointment.person), class: "img", aria: {hidden: true}, tabindex: "-1" %>
-    </div>
-    <h3 class="name"><%= link_to_if role_appointment.has_historical_account?, role_appointment.person.name, historic_appointment_path(role.historic_param, role_appointment.person), class: "govuk-link" %></h3>
-    <p class="term"><%= role_appointment.historical_account.try(:political_membership) %> <%= role_appointment.date_range %></p>
-  </div>
+<%= content_tag_for :div, role_appointment, class: 'govuk-grid-column-one-quarter', role: "listitem" do %>
+  <%= render "govuk_publishing_components/components/image_card", {
+    href: historic_appointment_path(role.historic_param, role_appointment.person),
+    image_src: role_appointment.person.image_url(:s216),
+    image_alt: role_appointment.person.name,
```

Comment:
The image alt text is a repeat of the heading text - could it be made a bit more descriptive? For example:

```suggestion
    image_alt: "Picture of #{role_appointment.person.name}",
```

Another good option would be to make the alt text field empty?

Reply:
I'd say we should blank the alt here, the only info conveyed by the picture is what the person looks like, if the alt isn't going to describe that (and obviously it can't since it's not hand crafted) there's no value to highlighting that there's a picture there at all, and repeating the name is just auditory clutter.

Reply:
I feel the word picture is redundant, it’s like “logo of” however happy remove the alt copy entirely - it’s not needed.